app/views/posts/show.html.erb

```diff
@@ -10,8 +10,8 @@
 
```

Comment:
- [ ] This change is intended for the new post page. When I visit the page on the broswer to create a post, it renders an error. Kindly ensure users can create a post, as this is important to meet the other requirements of the project.

  ![image](https://github.com/Xghayor/social-media/assets/81468675/910e68f4-e5a5-4412-bee6-592eca2de056)